### ISSUE TITLE
Parameters are duplicated when run mutiple times and python type hints are used.

### ISSUE THREAD

**Issue by thugcee** [dadadel/pyment#86]
Examples are for _numpy_ style and function `def foo(receiver_email: str)`:

When there is no given parameter in the doc string, it generates
```
receiver_email: str :
```
-- which is wrong, BTW.

On the second run it adds
```
receiver_email : str :
```
-- which is less wrong, but still not correct and duplicates previous one.

Changing manually to the correct form:
```
receiver_email : str
```
and running again causes addition of
```
receiver_email: str :
```

**Comment by Jawabiscuit** [comment]
I'd like to add that when the source is using type hinting and reST format the output formatted as google returns duplicate params as well.

An example patch:
```
-    
-    :param blendshapeNode: blendshape node with the desired target
-    :type blendshapeNode: str
-    :param str name: name of the desired target 
-    :returns: a target index
-    :rtype: int
+
+    Args:
+      blendshapeNode(str): blendshape node with the desired target
+      str: name: name of the desired target
+      name: 
+
+    Returns:
+      int: a target index
+
+    Raises:
+

```

**Comment by dadadel** [comment]
Hi @thugcee 
Actually, Pyment doesn't support type hints. So it consider `receiver_email: str` as the parameter name!
Note that I've just started to work on it (see issue #93).
@Jawabiscuit, this is an other problem, as types in reST are only managed using the `:type param:` in Pyment. This could also be managed in a future version.

**Comment by Jawabiscuit** [comment]
@dadadel, cool, it would really help transitioning large code bases to the more legible Google style docstrings! It would also help support the idea of transitioning in the first place if this feature were more accessible.

**Comment by dadadel** [comment]
FYI, type hints are now supported (in master, not yet released)!

**Comment by dadadel** [comment]
Furthermore, I think the duplication problem is related to this #95

**Comment by dadadel** [comment]
Hey @Jawabiscuit,
Can you tell me where did you see the usage of including the type in param statement for reST like `:param type name:`?

**Comment by dadadel** [comment]
As this issue's two problems are addressed with duplication in issue #95 and type hints in issue #93, I close this issue.
@Jawabiscuit, your "problem" of type inside param statement being specifically pointed in issue #85, please refer to that issue for tracking this.

**Comment by Jawabiscuit** [comment]
> Hey @Jawabiscuit,
> Can you tell me where did you see the usage of including the type in param statement for reST like `:param type name:`?

https://github.com/dadadel/pyment/issues/85#issuecomment-793209907
